src/main/java/kateryna/workshop_3/Task_2_longest_word.java

```diff
@@ -0,0 +1,31 @@
+package kateryna.workshop_3;
+
+/**
+ * Created by kate on 06.02.17.
+ */
+public class Task_2_longest_word {
+
+    private static void printTheLongestWord(String sentence){
+        if (sentence == "") {
```

Comment:
Better to compare strings with method equals()
== is used predominantly for primitives, I'll add an example into examples package with explanation on practice